crates/muvm/src/guest/bin/muvm-guest.rs

```diff
@@ -13,12 +7,36 @@ use muvm::guest::server::server_main;
 use muvm::guest::socket::setup_socket_proxy;
 use muvm::guest::user::setup_user;
 use muvm::guest::x11::setup_x11_forwarding;
+use muvm::utils::launch::GuestConfiguration;
+use nix::unistd::{Gid, Uid};
 use rustix::process::{getrlimit, setrlimit, Resource};
+use std::fs::File;
+use std::io::Read;
+use std::os::fd::AsFd;
+use std::panic::catch_unwind;
+use std::process::Command;
+use std::{cmp, env, fs, thread};
+
+const KRUN_CONFIG: &str = "KRUN_CONFIG";
 
 fn main() -> Result<()> {
     env_logger::init();
 
-    let options = options().run();
+    let config_path = env::args()
+        .nth(1)
+        .context("expected configuration file path")?;
+    let mut config_file = File::open(&config_path)?;
+    let mut config_buf = Vec::new();
+    config_file.read_to_end(&mut config_buf)?;
+    _ = fs::remove_file(config_path);
```

Comment:
Why not propagate errors here?

Reply:
Failing to remove this file is harmless, since it is on a tmpfs anyway. But i am not opposed to throwing an error here.

Reply:
Just seems like better hygiene when the file is supposed to always exist, since we just read it ^^

Reply:
If it's not *expected* to fail, we could just `.expect(...)`